.github/workflows/build_and_test.yml

```diff
@@ -148,7 +148,7 @@ jobs:
           # 3.12 will be ready when:
```

Comment:
Can you please remove the now irrelevant comments?

Reply:
Done!